src/com/strangeone101/elementumbot/command/LatencyCommand.java

```diff
@@ -0,0 +1,38 @@
+package com.strangeone101.elementumbot.command;
+
+import java.util.concurrent.Future;
+
+import de.btobastian.javacord.entities.message.Message;
+
+public class LatencyCommand extends CommandRunnable {
+
+	public LatencyCommand() {
+		super("latency");
+	}
+	
+	@Override
+	public void runCommand(Command command) {
+		if (!command.hasOppedPower()) return;
+		if (command.getArguments().length != 0) {
+			command.getOriginal().reply("Invalid usage! Usage is !latency");
+			return;
+		}
+		
+		long time = System.currentTimeMillis();
+		Future<Message> future = command.getOriginal().reply("Testing latency...");
+		Message message = null;
+		
+		try {
+			message = future.get();
+		} catch (Exception e) {
+			e.printStackTrace();
```

Comment:
Could you make this send a message like `"An error occurred while waiting: " + e.getLocalizedMessage()`?

Reply:
Roger roger